Source/utils/language.cpp

```diff
@@ -270,15 +270,19 @@ const std::string &LanguageTranslate(const char *key)
 
 bool HasTranslation(const std::string &locale)
 {
-	for (const char *ext : { ".mo", ".gmo" }) {
-		SDL_RWops *rw = OpenAsset((locale + ext).c_str());
+	if (locale == "en") {
+		return true;
+	}
+
+	constexpr std::array<char *, 2> Extensions { ".mo", ".gmo" };
+	return std::any_of(Extensions.cbegin(), Extensions.cend(), [locale](const std::string &extension) {

```

Comment:
[readability-use-anyofallof](https://clang.llvm.org/extra/clang-tidy/checks/readability-use-anyofallof.html) is enabled so I updated this while I was in the area. Personally I'm not a fan but it's one less squiggly line 🙃